source/documentation/appendix-docs/botor.md

```diff
@@ -1,35 +1,52 @@
 # Migrating to `botor`
 
 The new version of RStudio on the Analytical Platform uses `botor` to access
-our Amazon S3 data storage, replacing [`s3tools`](https://github.com/moj-analytical-services/s3tools). 
+our Amazon S3 data storage, replacing 
+[`s3tools`](https://github.com/moj-analytical-services/s3tools). 
 This guidance gives some hints on how to get going with `botor` and migrate 
 code that uses `s3tools`.
 
-This guidance assumes you have experience with `renv`. If you require a recap or guidance, please see either the [renv documentation from the platform guidance](https://github.com/moj-analytical-services/user-guidance/blob/dd896e73ed5be0dc42b9b38cd20e1e49e7cde560/source/documentation/tools/package-management.md#renv) or the [official renv documentation](https://rstudio.github.io/renv/articles/renv.html).
+This guidance assumes you have experience with `renv`. If you require a recap 
+or guidance, please see either the 
+[renv documentation from the platform guidance](https://user-guidance.services.alpha.mojanalytics.xyz/tools/package-management.html#renv) or the 
+[official renv documentation](https://rstudio.github.io/renv/articles/renv.html).
 
-# Table of Contents
+## Table of Contents
 * [Installation](#installation)
 * [Using botor](#usage)
-  * [Reading data](#reading-data)
-  * [Writing data](#writing-data)
-  * [Other useful functions](#other-useful-functions)
+    * [Reading data](#reading-data)
+    * [Writing data](#writing-data)
+    * [Other useful functions](#other-useful-functions)
 * [Migrating from s3tools](#migrating-from-s3tools)
-  * [read_using](#read_using)
-  * [s3_path_to_full_df](#s3_path_to_full_df)
-  * [write_df_to_csv_in_s3](#write_df_to_csv_in_s3)
-  * [download_file_from_s3](#download_file_from_s3)
-  * [botor examples and comparisons](#botor-examples)
+    * [Replacement functions](#replacement-functions)
+        * [`read_using`](#read-using)
+        * [`s3_path_to_full_df`](#s3-path-to-full-df)
+        * [`s3_path_to_preview_df`](#s3-path-to-preview-df)
+        * [`download_file_from_s3`](#download-file-from-s3)
+        * [`write_df_to_csv_in_s3`](#write-df-to-csv-in-s3)
+        * [`write_df_to_table_in_s3`](#write-df-to-table-in-s3)
+        * [`write_file_to_s3`](#write-file-to-s3)
+        * [`list_files_in_buckets`](#list-files-in-buckets)
+    * [botor examples and comparisons](#botor-examples)
```

Comment:
link doesn't currently work - this might simply be down to the backticks, though I haven't tested it

Reply:
The links don't work when viewing in github but do when building the site, which seems to handle them differently. Do you have a Mac? If so you can test the built site locally.